analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SonarAnalysisContextTest.cs

```diff
@@ -70,7 +70,7 @@ public TestSetup(string testCase, SonarDiagnosticAnalyzer analyzer, IEnumerable<
             new TestSetup("AnonymousDelegateEventUnsubscribe.cs", new AnonymousDelegateEventUnsubscribe()),
             // S2699 - TEST only
             new TestSetup("TestMethodShouldContainAssertion.NUnit.cs", new TestMethodShouldContainAssertion(),
-                TestMethodShouldContainAssertionTest.AdditionalTestReferences(NuGetMetadataReference.NUnit(Constants.NuGetLatestVersion))),
+                TestMethodShouldContainAssertionTest.WithTestReferences(NuGetMetadataReference.NUnit(Constants.NuGetLatestVersion)).References),    // ToDo: Reuse the entire builder in TestSetup
```

Comment:
NITPICK: Is the ToDO comment really needed?

Reply:
Yes, as I will not remember this intentino in the future when we bump into redesigning `SonarAnalysisCOntextTest`.